sugar_perception_node:
  ros__parameters:
    pc_topic: /camera_pan_tilt/points
    base_frame: panda_link0
    camera_frame: ""          # leave empty to use PointCloud2 header.frame_id
    tf_timeout_s: 0.5

    use_workspace_crop: true
    crop_x_min: 0.20
    crop_x_max: 0.90
    crop_y_min: -0.60
    crop_y_max: 0.60
    crop_z_min: 0.00
    crop_z_max: 1.20

    voxel_size: 0.005         # paper 0.5cm
    num_points: 4096
    sampling_method: random   # random | fps
    publish_debug_clouds: true

    debug_dir: /tmp/sugar_debug
    clear_debug_dir_on_start: true
    save_debug_npy: true
